GPU/Common/SplineCommon.cpp

```diff
@@ -361,16 +383,45 @@ static void _SplinePatchFullQuality(u8 *&dest, u16 *indices, int &count, const S
 	}
 }
 
+template <bool origNrm, bool origCol>
+static inline void SplinePatchFullQualityDispatch3(u8 *&dest, u16 *indices, int &count, const SplinePatchLocal &spatch, u32 origVertType, int quality, int maxVertices) {
+	bool origTc = (origVertType & GE_VTYPE_TC_MASK) != 0;
```

Comment:
This way, the compiler will generate separate specialized versions of the spline tesselation function for every combination of the orig* bools. Not sure it's actually worth it, unknownbrackets thought so though.